Code/Framework/AzCore/AzCore/Asset/AssetManager.cpp

```diff
@@ -27,6 +27,13 @@
 #include <utility>
 #include <AzCore/Serialization/ObjectStream.h>
 
+#define ENABLE_DEBUG_OUTPUT 0
+#if ENABLE_DEBUG_OUTPUT == 1
+#define DEBUG_OUTPUT(OUTPUT) AZ_Printf("AssetManager Debug", "%s\n", (OUTPUT).c_str())
```

Comment:
this feels generic enough that it might collide with defines in other libraries. Maybe change it to ASSET_DEBUG_OUTPUT?

Reply:
Good idea, will do